api/v1/controllers/samples.js

```diff
@@ -226,10 +234,23 @@ module.exports = {
 
         const j = jobWrapper.createJob(jobType.BULKUPSERTSAMPLES,
           wrappedBulkUpsertData, req);
-      } else if (featureToggles.isFeatureEnabled(constants.featureName)) {
-        redisModelSample.bulkUpsertSample(value);
       } else {
+        const bulkUpsertPromise =
+        featureToggles.isFeatureEnabled(constants.featureName) ?
```

Comment:
what is the reason for the change to either bulkUsert through the redisModel, or through the api?